src/devices/Si7021/Si7021.cs

```diff
@@ -9,8 +9,8 @@ namespace Iot.Device
 {
     public class Si7021 : IDisposable
     {
-        const byte si7021_cmd_read_temperature = 0xE3;
-        const byte si7021_cmd_read_humidity = 0xE5;
+        const byte Si7021_read_temperature_command = 0xE3;
```

Comment:
Instead of using underscore, can we have them use CammelCase instead?
```c#
private const byte ReadTemperatureCommand = 0xE3;
private const byte ReadHumidityCommand = 0xE5;
```
Also, we usually write the accessibility modifier explicitly.

Reply:
will change